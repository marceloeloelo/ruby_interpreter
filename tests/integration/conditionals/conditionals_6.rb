if ( a = true )
  puts ("bien")
  puts (a)
end

if ( "hola" )
  puts ("bien")
end

if ( a = "condition" )
  puts (a)
else
  puts ("mal")
end

if ( a = ( 1 == 1 ))
  puts (a)
else
  puts ("mal")
end

