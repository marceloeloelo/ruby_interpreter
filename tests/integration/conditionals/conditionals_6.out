bien
true
bien
condition
true
